sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -403,7 +403,6 @@ static bool PiDriverModuleProgramExtensionFound = false;
 // write (as it is OK for multiple threads to read the map without locking it).
 
 pi_result _pi_mem::addMapping(void *MappedTo, size_t Offset, size_t Size) {
```

Comment:
Update comments in the header that the "this->Mutex" should be locked before calling this.
Maybe, in fact, inline these functions and remove these methods

Reply:
Inlined these functions.